lab_3_keywords_textrank/main.py

```diff
@@ -215,7 +257,9 @@ def __init__(self) -> None:
         """
         Constructs all the necessary attributes for the adjacency matrix graph object
         """
-        pass
+        self._matrix = [[]]
+        self._positions = {}
+        self._position_weights = {}
 
     # Step 4.2
     def add_edge(self, vertex1: int, vertex2: int) -> int:
```

Comment:
Matrix is only supposed to contain zeros and ones, think of a matrix in math
Do not append vertices themselves in the matrix